docs/learn/lesson-2.mdx

```diff
@@ -187,7 +188,7 @@ Let's add the `TimePicker` input below the `DatePicker`:
     minTime="00:00"
     maxTime="23:45"
     step={15}
-    aria-describedby="startTime-error-message"
+    aria-describedby="start-time-error-message"
```

Comment:
I don't think these aria labels should be updated, they need to match the ids that will be introduced later on.